flink-examples/flink-java-examples/pom.xml

```diff
@@ -319,7 +319,30 @@ under the License.
 							</includes>
 						</configuration>
 					</execution>
-					
+
```

Comment:
I think the `collect()` example needs not be packaged into a JAR file. 

We package only examples as JAR files that compute something meaningful. Other examples are only includes as source.